test/mockBackend/activitySeries.js

```diff
@@ -0,0 +1,27 @@
+import { db } from '>/mockBackend/index'
+
+let nextId = 1
+export function generateActivitySeries (params = {}) {
+  if (!params.place) throw new Error('must provide place')
+  if (!params.activityType) {
+    // Auto-choose one based on place -> group
+    const place = db.orm.places.get({ id: params.place })
+    params.activityType = db.orm.activityTypes.get({ group: place.group }).id
```

Comment:
This will throw an error if there is not _exactly_ one activity type, I would suggest we create a `sample` method in the orm thing, so we can do:

```js
params.activityType = db.orm.activityTypes.sample({ group: place.group }).id
```

it would randomly select one, and if none, throw an error (as it would expect that there is at least one).

Reply:
good idea!

Reply:
I would leave this for another time though...